packages/flame/lib/src/geometry/rectangle_component.dart

```diff
@@ -33,6 +34,7 @@ class RectangleComponent extends PolygonComponent {
     super.paint,
     super.paintLayers,
     super.children,
+    super.scale,
```

Comment:
To keep consistent with the other components add it after `position` (usually it is after `size`, but that is not used in this constructor), and same with the default constructor.